protocol/v2/ssv/runner/runner_state_helpers.go

```diff
@@ -16,7 +17,16 @@ func getPreConsensusSigners(state *State, root [32]byte) []spectypes.OperatorID
 }
 
 func getPostConsensusSigners(state *State, root [32]byte) []spectypes.OperatorID {
-	sigs := state.PostConsensusContainer.Signatures[state.StartingDuty.(*spectypes.BeaconDuty).ValidatorIndex][hex.EncodeToString(root[:])]
+	var valIdx phase0.ValidatorIndex
+	switch state.StartingDuty.(type) {
+	case *spectypes.BeaconDuty:
+		valIdx = state.StartingDuty.(*spectypes.BeaconDuty).ValidatorIndex
+	case *spectypes.CommitteeDuty:
+		valIdx = state.StartingDuty.(*spectypes.CommitteeDuty).BeaconDuties[0].ValidatorIndex
```

Comment:
1. this gets only one validator index, there might be upto 500.
2. accessing 0 without checking the boundaries of the slice.

use a loop to collect all the ids. 
or just create two functions, one for committee and other for proposer runner.

Reply:
Added 2 separate helper functions